### ISSUE TITLE
Automatically update an API spec (periodically?) when it has been imported via an URL

### ISSUE THREAD

**Issue by mbouchenoire** [hopfront/hopfront#41]

**Comment by mbouchenoire** [comment]
I think the first iteration of this feature should be as follow:

HopFront displays an alert if the current spec is out of sync with the "remote" spec. The user can then decide himself if and _when_ he wants to upgrade. Automatically updating the spec sounds like a recipe for disaster (in the current state of things).

@remihrt @FabienWitick @AnthonyMde

**Comment by AnthonyMde** [comment]
Yes, I completely agree. Do we display the alert only for admin (if admin mode is enabled) or we let everyone see that the spec is stale?

**Comment by mbouchenoire** [comment]
> Yes, I completely agree. Do we display the alert only for admin (if admin mode is enabled) or we let everyone see that the spec is stale?

We should display it to everyone since it's not a guarantee that the admin is 1. logged in 2. using the app
